src/Build.UnitTests/BinlogVersionCompatibility/1.proj

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm guessing if the project body is inlined in the test class, no need to insert it here too